### PR TITLE
Release adjustments

### PR DESCRIPTION
**Pull request by Tristan1900**
### Change log description
- Add badge with latest release. It automatically updates the version when a release is published.
- Do not update `latest` Docker tag when release candidates are published.

---
Signed-off-by: wenqi <Wenqi.Mou@emc.com>

### CODE REVIEW

**Comment by Tristan1900** [comment]
@adrianmo Could you please take a look at the change in Travis? Thank you very much!

File: Makefile [hunk 1]
```diff
@@ -57,7 +57,9 @@ login:
 
 push: build login
 	docker push $(REPO):$(VERSION)
-	docker push $(REPO):latest
+	if [[ ${TRAVIS_TAG} =~ ^[0-9]+(\.[0-9]+)+(\.[0-9]+)$ ]]; then
```
Comment (adrianmo) [inline Makefile]: You don't need to create capture groups. Check out the regex pattern we are already using in the code: https://github.com/pravega/pravega-operator/blob/master/pkg/util/pravegacluster.go#L29

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
- [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![GoDoc](https://godoc.org/github.com/pravega/pravega-operator?status.svg)](https://godoc.org/github.com/pravega/pravega-operator) [![Build Status](https://travis-ci.org/pravega/pravega-operator.svg?branch=master)](https://travis-ci.org/pravega/pravega-operator) [![Go Report](https://goreportcard.com/badge/github.com/pravega/pravega-operator)](https://goreportcard.com/report/github.com/pravega/pravega-operator)
+ [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![GoDoc](https://godoc.org/github.com/pravega/pravega-operator?status.svg)](https://godoc.org/github.com/pravega/pravega-operator) [![Build Status](https://travis-ci.org/pravega/pravega-operator.svg?branch=master)](https://travis-ci.org/pravega/pravega-operator) [![Go Report](https://goreportcard.com/badge/github.com/pravega/pravega-operator)](https://goreportcard.com/report/github.com/pravega/pravega-operator)[![Version](https://img.shields.io/badge/release-0.4.0-green.svg)](https://github.com/pravega/pravega-operator/releases)
```
Comment (adrianmo) [inline README.md]: is it possible to add a dynamic version badge that is automatically updated when releases are published?

File: Makefile (hunk 1 above)
Comment (Tristan1900) [reply Makefile]: Got it, thanks!

File: README.md (hunk 2 above)
Comment (Tristan1900) [reply README.md]: Fixed! Thanks

**Pull request merged**